src/app/plugins/platform/JavaScript/templates/tasks/webpack.js

```diff
@@ -0,0 +1,16 @@
+const webpack = require('webpack');
+
+module.exports = function webpackBuild() {
+  return new Promise((resolve, reject) => {
+    const webpackConfig = require('../webpack.config')({'NODE_ENV': 'production'});
```

Comment:
I am of two minds about this line. On the one hand, it's nice to have this automatically (especially since it brings our tasks in line with `create-react-app` and friends) - but on the other, this can lengthen the turnaround time for dev site builds. On projects with a lot of images, there will be a lot of optimization running before the minifier can even trigger.

I don't think I feel super strongly about this, so I'd like to hear other opinions.

Reply:
I am of the opinion that a build task should produce artifacts that are fully compiled, minified, and ready for production. If there's a use case for building assets outside of the webpack-dev-server task that doesn't minify them, my vote would be to add a `gulp dev` or `gulp build_dev` task.

Reply:
I can get behind that. Let's keep this and see if the need arises for a dev build.